src/server/tiered_storage.cc

```diff
@@ -560,7 +559,7 @@ size_t TieredStorage::ReclaimMemory(size_t goal) {
 }
 
 bool TieredStorage::ShouldStash(const PrimeValue& pv) const {
-  auto disk_stats = op_manager_->GetStats().disk_stats;
+  const auto& disk_stats = op_manager_->GetStats().disk_stats;
```

Comment:
GetStats() returns by value so that was free 🙂

Reply:
oh